### ISSUE TITLE
Incorporate "Advanced UI" docs elsewhere in flattened UI categories

### ISSUE THREAD

**Issue by parlough** [flutter/website#9012]
Items get lost under "Advanced UI". They should either be moved to other existing (or new) UI categories, or moved to references when appropriate.

Most of this is probably blocked by https://github.com/flutter/website/issues/9011.

**Comment by parlough** [comment]
Duplicate of https://github.com/flutter/website/issues/8997 - Oops 😄
